### PR TITLE
ENH add covtype gisette sector scale usps sensit

### PR DESCRIPTION
**Pull request by QB3**

### CODE REVIEW

File: libsvmdata/datasets.py [hunk 1]
```diff
@@ -33,19 +35,24 @@
     'rcv1_train': 'binary/rcv1_train.binary.bz2',
     'real-sim': 'binary/real-sim.bz2',
     'sector_train': 'multiclass/sector/sector.bz2',
+    'sector_scale': 'multiclass/sector/sector.scale.bz2',
```
Comment (mathurinm) [inline libsvmdata/datasets.py:37]: can you keep the dict keys them in alphabetical order ? (so move it one line up)

File: libsvmdata/datasets.py [hunk 2]
```diff
@@ -33,19 +35,24 @@
...
 
 N_FEATURES = {
     'aloi': 128,
     'bodyfat': 14,
     'connect-4': 126,
+    'covtype': 54,
     'dna': 180,
     'eunite2001': 16,
     'finance': 4_272_227,
+    'gisette': 5000,
```
Comment (mathurinm) [inline libsvmdata/datasets.py]:
```suggestion
    'gisette': 5_000,
```

File: libsvmdata/datasets.py [hunk 3]
```diff
@@ -65,6 +72,9 @@
     'smallNORB': 18_432,
     'url': 3_231_961,
     'webspam': 16_609_143,
+    'usps': 7_291,
```
Comment (mathurinm) [inline libsvmdata/datasets.py:76]: can you move them in the dict so that the keys remains sorted alphabetically?

**Comment by mathurinm** [comment]
Thanks, it works fine on my side, can you just address the 2 comments and then we merge ?

**Comment by QB3** [comment]
everything should be fine now!

**Pull request merged**